Copyediting (mostly) on Aftershock's README.md (#47785)
CleverRaven/Cataclysm-DDA: data/mods/Aftershock/README.md

# Design Document

> * Aftershock is being developed into a full conversion mod set in a mostly-abandoned extrasolar colony named Salus IV.  The player has arrived on Salus for various reasons (depending on their profession).  Eventually, we hope to have multiple potential end game situations, ranging from finding what you came here for and leaving the planet with enough money to retire, to more-secret endings.  There will no longer be zombies; instead, there are a variety of alien and human-based enemies.  Toe-to-toe combat will likely be more deadly in Aftershock, with more emphasis on picking your battles and only fighting when absolutely necessary.  Special locations may have a different combat aesthetic; for example, you may enter a dungeon-like zone with hordes of small weak enemies that can't be avoided.  As part of this concept, in which different situations require different solutions, we have implemented bionic slots but made the process of changing out CBMs much easier with less chance of failure.  This is to encourage players to change their loadout based on where they are going and what they are doing.

Author: Regarding "changing out CBMs" - is this already implemented?

Reviewer: Yes, mostly as a modification of Autodoc skill multipliers

### npcs

> Contains json data for npcs, factions, and design document for any new factions, specific new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colony separate from Corporate and Earthgov influences.  The Whately Clan are a possibly-exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know.  Mutant NPCs are also located in this folder.

Author: ```suggestion
Contains json data for npcs and factions, which are currently the new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colony separate from Corporate and Earthgov influences.  The Whately Clan are a possibly-exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know.  Mutant NPCs are also located in this folder.
```
I can't find a faction design document.

Reviewer: Its in-progress draft is in here: https://github.com/John-Candlebury/Cataclysm-DDA/wiki

Author: Should I put in a link?

Reviewer: Probably not yet.
